cirq-core/cirq/neutral_atoms/neutral_atom_devices_test.py

```diff
@@ -136,11 +136,11 @@ def with_qubits(self, new_qubits):
         d.validate_operation(cirq.CCX.on(*d.qubit_list()[0:3]))
     with pytest.raises(ValueError, match="are too far away"):
         d.validate_operation(cirq.CZ.on(cirq.GridQubit(0, 0), cirq.GridQubit(2, 2)))
-    with pytest.raises(ValueError, match="Too many Z gates in parallel"):
+    with pytest.raises(ValueError, match="Unsupported operation"):
         d.validate_operation(cirq.parallel_gate_op(cirq.Z, *d.qubits))
-    with pytest.raises(ValueError, match="Bad number of XY gates in parallel"):
+    with pytest.raises(ValueError, match="Unsupported operation"):
         d.validate_operation(cirq.parallel_gate_op(cirq.X, *d.qubit_list()[1:]))
-    with pytest.raises(ValueError, match="ParallelGate over MeasurementGate is not supported"):
+    with pytest.raises(ValueError, match="Unsupported operation"):
```

Comment:
Are we at all worried that these errors are a little less informative now ?

Reply:
I wouldn't worry so much, especially because cirq/neutral_atom is also not a vendor maintained device and hence is not expected to be used widely.

Reply:
Fair point. @dstrain115 can you confirm that this is a less active vendor ?

Reply:
Or maybe @c-poole ?